composer.json

```diff
@@ -28,6 +29,7 @@
     "require-dev": {
         "symfony/http-foundation": "^3.1@dev || ^4.0@dev",
 
+        "alcaeus/mongo-php-adapter": "^1.1",
```

Comment:
i would recommend to not mix mongodb and phpcr support into one pull request. it will be easier to review if its first one and then the other once that goes through.